kak-lsp.toml

```diff
@@ -69,6 +69,21 @@
 command = "vscode-css-languageserver"
 args = ["--stdio"]
 
+
+# See https://scalameta.org/metals/docs/integrations/new-editor
+[language_server.metals]
+filetypes = ["scala", "sbt", "sc"]
+roots = ["build.sbt", ".scala-build"]
+command = "metals"
+args = ["-Dmetals.extensions=false"]
+settings_section = "metals"
+
```

Comment:
can we delete the empty line? To help `<a-i>p` ;)

Reply:
Sure, done